apache/cloudstack
# Added support for removing unused port groups on VMWare

### Description

This PR adds a global setting (vmware.cleanup.port.groups) to toggle removing unused port groups from VMware hypervisor hosts.









Fixes: #3779 





### Types of changes


### Feature/Enhancement Scale or Bug Severity

#### Feature/Enhancement Scale


#### Bug Severity



### Screenshots (if appropriate):


### How Has This Been Tested?




This has been manually tested, with the below steps:
* Set global setting (vmware.cleanup.port.groups) to true / false.
* Create an instance on a new network.
* Check network config for hosts in vSphere client for the newly created port group.
* Destroy and expunge the instance.
* Destroy the router.
* Check network config for hosts in vSphere client if the port group has been removed or not, depending on global setting (vmware.cleanup.port.groups)
